integration-tests/plugins/index.js

```diff
@@ -22,6 +22,7 @@ module.exports = (on) => {
     stubVerifyToken: (active = true) => tokenverification.stubVerifyToken(active),
     stubLoginPage: auth.redirect,
     stubAuthUsernameSearch: auth.stubAuthUsernameSearch,
+    stubAuthGetUsername: auth.stubAuthGetUsername,
```

Comment:
we were previously using username search for the get username call, but now have had to split out